test/test_gradio_component_cli.py

```diff
@@ -106,39 +106,35 @@ def test_show(capsys):
     assert "Dataset" not in stdout
 
 
-@pytest.mark.xfail
-@pytest.mark.parametrize("template", ["Audio", "Video", "Image", "Textbox"])
-def test_build(template, tmp_path):
-    _create(
-        "TestTextbox",
-        template=template,
-        directory=tmp_path,
-        overwrite=True,
-        install=True,
-        configure_metadata=False,
-    )
-    _build(tmp_path, build_frontend=True)
-    template_dir: Path = (
-        tmp_path.resolve() / "backend" / "gradio_testtextbox" / "templates"
-    )
-    assert template_dir.exists() and template_dir.is_dir()
-    assert list(template_dir.glob("**/index.js"))
-    assert (tmp_path / "dist").exists() and list((tmp_path / "dist").glob("*.whl"))
-
-
-def test_install(tmp_path):
-    _create(
-        "TestTextbox",
-        template="Textbox",
-        directory=tmp_path,
-        overwrite=True,
-        install=False,
-        configure_metadata=False,
-    )
-
-    assert not (tmp_path / "frontend" / "node_modules").exists()
-    _install(tmp_path)
-    assert (tmp_path / "frontend" / "node_modules").exists()
+@pytest.mark.parametrize("template", ["Audio", "Image"])
+def test_build(template):
+    pnpm_lock = Path(__file__).parent / ".." / "pnpm-lock.yaml"
+    pnpm_copy = Path(__file__).parent / ".." / "pnpm-lock-copy.yaml"
+    shutil.copy(str(pnpm_lock), str(pnpm_copy))
+    # Using the js/preview/test directory will use the workspace code
```

Comment:
We'll always install the local `@gradio/` js packages so this should pass on release PRs